layouts/home.html

```diff
@@ -72,111 +72,108 @@ <h1 class="text-5xl font-medium">{{ .Title }}</h1>
       </nav>
     </div>
     <div class="lg:px-11 w-full flex flex-col gap-3">
-      <h2 class="text-5xl font-medium" id="operate">Learn</h2>
+      <h2 class="text-5xl font-medium" id="operate">Get Started</h2>
         {{ partial "docs-section.html" (dict
-          "Title" "Develop with Redis"
-          "Description" "Learn how to use the Redis in-memory data store."
-	  "ButtonLink" "./develop"
-          "ButtonLabel" "Learn more"
-          "LinksLeftTitle" "Featured content"
+          "Title" "Quickly set up Redis in the cloud"
+          "Description" "Free Redis Cloud account managed by Redis or on AWS, GCP, or Azure"
+	  "ButtonLink" "./operate/rc/rc-quickstart/"
+          "ButtonLabel" "Redis Cloud quickstart setup guide" 
           "LinksLeft" (slice
-            (dict "Text" "Vector search" "URL" "./develop/interact/search-and-query/query/vector-search")
-            (dict "Text" "Data structure store" "URL" "./develop/get-started/data-store/")
-	    (dict "Text" "Document database" "URL" "./develop/get-started/document-database/")
+            (dict "Text" "High availability & replication guide" "URL" "./operate/rc/databases/configuration/high-availability/")
           )
         ) }}
         {{ partial "docs-section.html" (dict
-          "Title" "Libraries and tools"
-          "Description" "Learn about libraries and tools available for Redis."
-          "ButtonLink" "./integrate"
-          "ButtonLabel" "Browse tools"
-          "LinksLeftTitle" "Featured libraries"
-          "LinksLeft" (slice
-            (dict "Text" "RedisVL" "URL" "./integrate/redisvl/")
-            (dict "Text" "RedisOM for Python" "URL" "./integrate/redisom-for-python")
-            (dict "Text" "Ingest" "URL" "./integrate/redis-data-integration/")
-            (dict "Text" "Write-behind" "URL" "./integrate/write-behind/quickstart/write-behind-guide/")
-          )
-        ) }}
+          "Title" "Build modern real-time applications with Redis"
+          "Description" "Install and use Redis Stack on Linux, macOS, Windows, or Docker"
+	  "ButtonLink" "./operate/oss_and_stack/install/install-stack/"
+          "ButtonLabel" "Redis Stack install guide"  
+	"LinksLeft" (slice
+            (dict "Text" "Redis for AI docs and examples" "URL" "./develop/ai/")
+          )  
+	) }}
         {{ partial "docs-section.html" (dict
-          "Title" "Redis Data Integration"
-          "Description" "Redis Data Integration keeps Redis in sync with the primary database in near real time."
-          "ButtonLink" "./integrate/redis-data-integration"
-          "ButtonLabel" "Learn more"
-          "LinksLeftTitle" "LEARN MORE"
+         "Title" "Optimize Redis for high-scale apps"
+          "Description" "Get the Redis Insight GUI on AWS EC2, Docker, Kubernetes, and desktop"
+	  "ButtonLink" "./operate/redisinsight/install/"
+          "ButtonLabel" "Redis Insight installation guide"  
```

Comment:
```suggestion
          "ButtonLabel" "Redis Insight install guide"  
```
I think we should be consistent with the use of "install guide" instead of "installation guide", but would personally prefer this:

```suggestion
          "ButtonLabel" "Install Redis Insight"  
```